controllers/util/storageconsumer.go

```diff
@@ -308,6 +309,30 @@ func GetStorageConsumerDefaultResourceNames(
 	return defaults
 }
 
+func GenerateCsiRbdProvisionerCephClientName(uid types.UID) string {
+	return fmt.Sprintf("%s-g%s-%v", csiRbdProvisionerCephUserKey, csiCephUserCurrGen, uid)
+}
+
+func GenerateCsiRbdNodeCephClientName(uid types.UID) string {
+	return fmt.Sprintf("%s-g%s-%v", csiRbdNodeCephUserKey, csiCephUserCurrGen, uid)
+}
+
+func GenerateCsiCephFsProvisionerCephClientName(uid types.UID) string {
+	return fmt.Sprintf("%s-g%s-%v", csiCephFsProvisionerCephUserKey, csiCephUserCurrGen, uid)
+}
+
+func GenerateCsiCephFsNodeCephClientName(uid types.UID) string {
+	return fmt.Sprintf("%s-g%s-%v", csiCephFsNodeCephUserKey, csiCephUserCurrGen, uid)
+}
+
+func GenerateCsiNfsProvisionerCephClientName(uid types.UID) string {
+	return fmt.Sprintf("%s-g%s-%v", csiNfsProvisionerCephUserKey, csiCephUserCurrGen, uid)
+}
+
+func GenerateCsiNfsNodeCephClientName(uid types.UID) string {
+	return fmt.Sprintf("%s-g%s-%v", csiNfsNodeCephUserKey, csiCephUserCurrGen, uid)
+}
```

Comment:
Please make sure all of these function do not relay on global state. Pass in the required generation from the call site